wdio.conf.js

```diff
@@ -11,7 +11,8 @@ const config = {
   seleniumDocker: {
     enabled: !process.env.TRAVIS,
   },
-
+  
+  host: '0.0.0.0',
```

Comment:
why?

Reply:
This is temporary until I can get all of the parallelization changes propagated.  It is the default host used in webdriver.